pages/en/committee.markdown

```diff
@@ -19,7 +19,7 @@ Pauline Karega is a bioinformatics student and researcher working on evolutionar
 ## Margaret Wanjiku
 ![Add Profile Photo](https://sbcf.fr/wp-content/uploads/2018/03/sbcf-default-avatar.png)
 
-<!--add bio here-->
+Margaret serves the role of secretary to the BHKI steering committee. She is a bioinformatician with a keen interest in omics and big data analysis.
```

Comment:
@Megmugure  this same info is in the #18 pull? Maybe incorporate the suggestions from @bkmgit

Reply:
I accidentally committed it twice, I was afraid that there might be some conflicts. I hope it can be rectified.

Reply:
Can be fixed when merging, so should be ok.